protos/ensembl/production/metadata/grpc/ensembl_metadata.proto

```diff
@@ -413,7 +412,7 @@ message GenomeDatatypeRequest {
 Genome info filter used by Compara (EA-1090)
  */
 message GenomeInfoRequest {
-  string ensembl_name = 1;     // Mandatory
+  string production_name = 1;     // Mandatory
```

Comment:
We should have kept the public facing "ensembl_name" because in the long run the production_name will disappear (it's here only to match current Hive Pipelines requirements to have such a value)

Reply:
Do you mean keeping both `ensembl_name` and `production_name`?

Reply:
No, I mean that `production_name` it definitely internal. Keep it as such. Don't bother with my late thoughts.